### DISCUSSION TITLE
Convolutional block DOES NOT seem to support Conv1D?

### DISCUSSION THREAD

**Discussion by istavnit** [keras-team/autokeras#1345] [General] [answered]
Is this a deliberate omission?

**Comment by haifeng-jin** [comment, marked as answer]
I think the ConvBlock class support 1D data.
If it doesn't, we have an example here: https://github.com/keras-team/autokeras/blob/master/examples/automodel_with_cnn.py

**Reply by istavnit** [reply]
I am sorry it does support it - as long as I apply np.expand_dims to it. It requires a 3+d tensor which is different than its Keras Conv1D counterpart.
Thank you for the answer.
